dependency_check_suppressions.xml

```diff
@@ -95,6 +109,13 @@
     <vulnerabilityName>CVE-2022-1799</vulnerabilityName>
     <vulnerabilityName>CVE-2022-2390</vulnerabilityName>
   </suppress>
+  <suppress until="2023-01-31Z">
+    <notes><![CDATA[
+   file name: sdk-utils-2.1.1.aar
```

Comment:
Seems SDK Utils needs an update.

Reply:
@leila-gu Yes, will create ticket for it